release_task.rb

```diff
@@ -184,4 +186,11 @@ def define_version_update_task
       end
     end
   end
+
+  def define_release_task
+    desc "Release"
+    task release: [
+      "release:version:update"
+    ]
```

Comment:
How about executing this in `define` without creating a new method?

Executing this in `ReleaseTask#define` isn't strange.